### PR TITLE
New Adapter: Zeta Global SSP

### PR DESCRIPTION
**Pull request by abermanov-zeta**
Docs: https://github.com/prebid/prebid.github.io/pull/4464

### CODE REVIEW

File: src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                .extracting(httpRequest -> mapper.readValue(httpRequest.getBody(), BidRequest.class))
+                .containsOnly(bidRequest);
+    }
+
+    @Test
+    public void makeHttpRequestsShouldMakeOneRequestForMultipleImps() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                identity(),
+                requestBuilder -> requestBuilder.imp(Arrays.asList(
```
Comment (CTMBNara) [inline src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java:68]: Nitpick: we prefer to import static methods in tests.

File: src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        final BidRequest bidRequest = givenBidRequest(identity());
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = bidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(httpRequest -> mapper.readValue(httpRequest.getBody(), BidRequest.class))
+                .containsOnly(bidRequest);
```
Comment (CTMBNara) [inline src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java:60]: `hasSize` && `containsOnly` can be replaced with `containsExactly`. Please, refactor this.
```
assertThat(...)
     .extracting(...)
     .containsExactly(bidRequest);
```

File: src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnBannerBidByDefault() throws JsonProcessingException {
+        // given
+        final BidderCall<BidRequest> httpCall = givenHttpCall(
+                BidRequest.builder().imp(singletonList(Imp.builder().id("123").build())).build(),
```
Comment (CTMBNara) [inline src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java:128]: Can be simplified using `givenBidRequest`.  Please, refactor this and other similar places.

**Review by CTMBNara — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidder.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        return bidsFromResponse(bidRequest, bidResponse);
+    }
+
+    private static List<BidderBid> bidsFromResponse(BidRequest bidRequest, BidResponse bidResponse) {
+        return bidResponse.getSeatbid().stream()
+                .filter(Objects::nonNull)
+                .map(SeatBid::getBid)
+                .filter(Objects::nonNull)
+                .flatMap(Collection::stream)
+                .map(bid -> BidderBid.of(bid, getBidType(bid, bidRequest.getImp()), bidResponse.getCur()))
```
[the remark this answers is not in the document]
Comment (abermanov-zeta) [reply src/main/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidder.java]: Done, thank you

File: src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java (hunk 1 above)
Comment (abermanov-zeta) [reply src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java]: Done, thank you

File: src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java (hunk 2 above)
Comment (abermanov-zeta) [reply src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java]: Refactored, thank you

File: src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java (hunk 3 above)
Comment (abermanov-zeta) [reply src/test/java/org/prebid/server/bidder/zeta_global_ssp/ZetaGlobalSspBidderTest.java]: Refactored, thank you

File: src/main/resources/bidder-config/zeta_global_ssp.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+adapters:
+  zeta-global-ssp:
```
[the remark this answers is not in the document]
Comment (abermanov-zeta) [reply src/main/resources/bidder-config/zeta_global_ssp.yaml:2]: Yes, it's intended. `modifying-vast-xml-allowed` is `true` by default in Prebid Server Java, right?

[the remark this answers is not in the document]
Comment (CTMBNara) [reply src/main/resources/bidder-config/zeta_global_ssp.yaml:2]: @abermanov-zeta true, but some host companies can configure it to `false` as their default. Is it OK for you?

[the remark this answers is not in the document]
Comment (abermanov-zeta) [reply src/main/resources/bidder-config/zeta_global_ssp.yaml:2]: Yes, it's ok

**Review by CTMBNara — APPROVED** [review]

**Comment by abermanov-zeta** [comment]
@SerhiiNahornyi Could you please advise when the next release is going to happen? Thank you.

**Pull request merged**